docker.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
+const {config} = bedrock;
+const path = require('path');
+const process = require('process');
+
+// load configs from /etc/veres-one
+config.paths.config = path.join('/etc', 'veres-one');
+
+// ensure that files in /etc/veres-one can require files in app directory
```

Comment:
Not sure exactly what's going on here, but I notice that the WORKDIR is not being set in the dockerfile, is that a possible remedy to not have to do this?

Reply:
That's not a remedy. The issue is that (I think) we do a require in /etc/veres-one/ in the docker image and since there is no node_modules directory in /etc/veres-one/ and /etc/, the require fails for any config file in /etc/veres-one.

That said, I don't remember if I tried setting WORKDIR in the Dockerfile... I can try that when I get back around to rebuilding the docker images.

Reply:
I did check w/ @dlongley on the magic algorithm that node uses to construct the node_modules search path, and I didn't think it took the current working directory into account (because that could lead to terrible security vulnerabilities... "Hey friend, why don't you cd to /home/msporny and run that /usr/bin/ponies tool that prints out a pony... HAHAHA, I HAVE ROOT NOW! (because I tricked you into running from my directory, which has a compromised node_modules directory)".

... but I can try setting WORKDIR to see if node has that sort of vulnerability.

Reply:
> ... but I can try setting WORKDIR to see if node has that sort of vulnerability.

I tried setting WORKDIR, it didn't work.